code/game/objects/structures/signs/signs_interactive.dm

```diff
@@ -82,3 +79,75 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/delamination_counter, 32)
 			. += span_info("Good work!")
 		if(11 to INFINITY)
 			. += span_info("Incredible!")
+
+GLOBAL_LIST_EMPTY(map_collision_counters)
+
+/obj/structure/sign/collision_counter
+	name = "incident counter"
+	sign_change_name = "Indicator board- Tram incidents"
+	desc = "A display that indicates how many tram related incidents have occured today."
+	icon_state = "tram_hits"
+	is_editable = TRUE
+	var/hit_count = 0
+	var/tram_id = TRAM_LIFT_ID
+	var/datum/weakref/tram_ref
+
+/obj/structure/sign/collision_counter/proc/find_tram()
+	for(var/obj/structure/industrial_lift/tram/tram as anything in GLOB.lifts)
+		if(tram.lift_id != tram_id)
+			continue
+		tram_ref = WEAKREF(tram)
+		break
+
+/obj/structure/sign/collision_counter/Initialize(mapload)
+	. = ..()
+
+	return INITIALIZE_HINT_LATELOAD
+
+/obj/structure/sign/collision_counter/LateInitialize()
+	. = ..()
+	find_tram()
+	var/obj/structure/industrial_lift/tram/tram = tram_ref?.resolve()
```

Comment:
like couldn't find_tram just return the tram to use?

Reply:
I'll take a look, I was just reusing the other tram code.